libqtile/backend/wayland/window.py

```diff
@@ -686,10 +698,67 @@ def enable_fullscreen(self) -> None:
     def disable_fullscreen(self) -> None:
         self.fullscreen = False
 
+    def reparent(self, layer: SceneTree) -> None:
+        # already in the right layer
+        if self.tree_layer is layer:
+            return
+        self.tree_layer = layer
+        self.container.node.reparent(layer)
+
     @expose_command()
-    def bring_to_front(self) -> None:
+    def keep_below(self) -> None:
+        self.reparent(self.core.keep_below_window_tree)
+
+    @expose_command()
+    def keep_above(self) -> None:
+        self.reparent(self.core.keep_above_window_tree)
+
+    @expose_command()
+    def move_up(self, force: bool = False) -> None:
+        if force and self.tree_layer is self.core.keep_below_window_tree:
+            self.reparent(self.get_new_layer(self._float_state))
```

Comment:
@elParaguayo does move_up or down with force also remove it from bring_to_front layer?

Reply:
x11 doesn't have a `bring_to_front` layer. In x11, `bring_to_front` ignores layers and brings the window to the front. Once it loses focus, it should be restacked according to its layer.

The behaviour of `force` is as follows. If this is `True`, `move_up` will move the window out of the `kept_below` layer group, and `move_down` will move the window out of the `kept_above` layer group. If `False`, the windows can only move within their respective layer groups.